lib/flying_shuttle/route_service.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+require 'open3'
+require_relative 'mixins/weave_helper'
+require_relative 'mixins/logging'
+
+module FlyingShuttle
+  class RouteService
+    include Logging
+    include WeaveHelper
+
+    REGION_LABEL = FlyingShuttle::REGION_LABEL
+    IPTABLES_DNAT = 'iptables %s -w -t nat -p tcp -d %s --dport 10250  -j DNAT --to-destination %s:10250'
```

Comment:
The first `%s` is actually two things, it combines the action and chain name. maybe consider splitting that.

Reply:
I think I'm going to keep it as one.. but I'll refactor these as "named parameters" for easier reading.